lib/cylc/prompt.py

```diff
@@ -19,12 +19,30 @@
 import sys
 
 from cylc.cfgspec.globalcfg import GLOBAL_CFG
+from cylc.gui.util import get_icon
```

Comment:
Need to remove this line as well. (`get_icon` is not used in this module, and it is causing another *cannot open display* failure.)